assets/blocks/course-progress-block/course-progress-edit.js

```diff
@@ -20,6 +20,7 @@ import {
 import { COURSE_STATUS_STORE } from '../course-outline/status-preview/status-store';
 import CourseProgressSettings from '../editor-components/course-progress-settings';
 import ToggleLegacyCourseMetaboxesWrapper from '../toggle-legacy-course-metaboxes-wrapper';
+import CourseProgress from '../../shared/blocks/course-progress';
```

Comment:
I think that the current directory structure is a bit confusing. We have the block under `shared` while its settings are under `editor-components`. We can improve this by:
- Have both the components and its settings under the same directory.
- Use a single import to import both.

Reply:
You're right! I was trying to avoid duplicated code in different assets, but it wasn't well organized.
I updated it in this commit: https://github.com/Automattic/sensei/pull/4149/commits/3304585ae7b3ec90e6ef5f08e5006f23fb381f64

It'll duplicate a few lines in different assets for now. In the future, we have to think again about the assets structure.